context.go

```diff
@@ -24,6 +24,11 @@ func Unhandled(request *Message) {
 
 // Defines how incoming requests are dispatched to handler functions.
 type Context struct {
+
+	// The WebSocket subprotocol that this blip context is constrained to.  Eg: BLIP_3+CBMobile_2
+	// Client request must indicate that it supports this protocol, else WebSocket handshake will fail.
+	WebSocketSubProtocol string
+
 	HandlerForProfile map[string]Handler // Handler function for a request Profile
```

Comment:
OMG the types don't line up! ;) Need to run `go format`.

Reply:
Haha, actually this is what `gofmt` does when there is an empty space between "sections" in a struct.  I wanted to more comments on `WebSocketSubProtocol` than would fit to the right of the field name, so I put it into it's own section.